webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -77,6 +77,9 @@
                 {% endfor %}
             </tbody>
         </table>
+        <em>
```

Comment:
I.e. I think it should be: `<p class="footnote">` and something like `p.footnote { font-style: italic; }`